src/views/GovernanceReviewTeam/GRBReview/PresentationLinksForm/index.tsx

```diff
@@ -0,0 +1,299 @@
+import React from 'react';
+import { Controller } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+  Button,
+  Fieldset,
+  Form,
+  FormGroup,
+  Grid,
+  Icon,
+  TextInput
+} from '@trussworks/react-uswds';
+import {
+  SystemIntakeGRBPresentationLinks,
+  SystemIntakeGRBPresentationLinksInput,
+  useSetSystemIntakeGRBPresentationLinksMutation
+} from 'gql/gen/graphql';
+
+import { useEasiForm } from 'components/EasiForm';
+import FileInput from 'components/FileInput';
+import Alert from 'components/shared/Alert';
+import HelpText from 'components/shared/HelpText';
+import IconLink from 'components/shared/IconLink';
+import Label from 'components/shared/Label';
+import { TabPanel, Tabs } from 'components/Tabs';
+import useMessage from 'hooks/useMessage';
+import { fileToBase64File } from 'utils/downloadFile';
+import { SetGRBPresentationLinksSchema } from 'validations/grbReviewSchema';
+
+import './index.scss';
+
+type PresentationLinkFields = Omit<
+  SystemIntakeGRBPresentationLinksInput,
+  'systemIntakeID'
+>;
+
+type PresentationLinksFormProps = {
+  id: string;
+  // Pass presentation link to form if editing
+  grbPresentationLinks?: SystemIntakeGRBPresentationLinks;
+};
+
+/**
+ * Form to add or edit GRB review presentation links
+ */
+const PresentationLinksForm = ({
+  id,
+  grbPresentationLinks
+}: PresentationLinksFormProps) => {
+  const { t } = useTranslation('grbReview');
+  const { showMessage, showMessageOnNextPage } = useMessage();
+  const history = useHistory();
+
+  const [setPresentationLinks] = useSetSystemIntakeGRBPresentationLinksMutation(
+    { refetchQueries: ['GetSystemIntake'] }
+  );
+
+  const {
+    register,
+    handleSubmit,
+    control,
+    formState: { errors, isValid, isDirty, defaultValues }
+  } = useEasiForm<PresentationLinkFields>({
+    resolver: yupResolver(SetGRBPresentationLinksSchema),
+    defaultValues: {
+      recordingLink: grbPresentationLinks?.recordingLink,
+      recordingPasscode: grbPresentationLinks?.recordingPasscode,
+      transcriptLink: grbPresentationLinks?.transcriptLink
+    }
+  });
+
+  const formType: 'add' | 'edit' = grbPresentationLinks ? 'edit' : 'add';
+
+  const grbReviewPath = `/it-governance/${id}/grb-review`;
+
+  /**
+   * Returns true if both recordingLink and presentationDeckFileData fields have errors
+   */
+  const formIsEmpty =
+    !!errors?.recordingLink && !!errors?.presentationDeckFileData;
+
```

Comment:
The naming of this variable is a bit unintuitive if it's checking for errors/saying it's empty